test/integration/nodebalancers_test.go

```diff
@@ -85,11 +86,15 @@ func setupNodeBalancer(t *testing.T, fixturesYaml string) (*linodego.Client, *li
 	t.Helper()
 	var fixtureTeardown func()
 	client, fixtureTeardown := createTestClient(t, fixturesYaml)
+	client, firewall, firewallTeardown, err := setupFirewall(t, []firewallModifier{}, "fixtures/TestNodeBalancer_Create")
 	createOpts := linodego.NodeBalancerCreateOptions{
 		Label:              &label,
```

Comment:
`setupNodeBalancer` function is shared with different test cases, so we can't use the static string here as the fixture name. And, I am not sure if calling 2 setup functions here will work since they may override the fixture files generated by each other...

Reply:
Fixing this now!